src/content/configuration/dev-server.mdx

```diff
@@ -841,8 +841,8 @@ Provides the ability to execute a custom function when webpack-dev-server starts
 module.exports = {
   //...
   devServer: {
-    onListening: function (server) {
-      const port = server.address().port;
+    onListening: function (devServer) {
+      const port = devServer.server.address().port;
```

Comment:
Should we check the existence of `devServer` first? https://github.com/webpack/webpack-dev-server/blob/974ce25669ef6a4f55e8a7576fc140bc7ccb55f1/test/server/onListening-option.test.js#L18

Reply:
Not sure, it is unlikely to be `undefined`. But no harm in checking.

https://github.com/webpack/webpack-dev-server/blob/bbbcd7e7de70c28369ef70bed5c1e940c875fb2d/lib/Server.js#L1780

Reply:
/cc @alexander-akait what do you think?